.github/workflows/npm-publish-alpha.yml

```diff
@@ -10,13 +10,13 @@ jobs:
   publish-npm-alpha:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Get short SHA
-        id: sha
-        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+        id: set_sha
+        run: echo "SHA_SHORT=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
```

Comment:
wow ... `>> $GITHUB_ENV` is quite strange. Wondering why this is needed?

Reply:
This comes from this deprecation note: 

![image](https://github.com/kbss-cvut/s-forms/assets/57750426/1c946659-10bc-4c71-bc34-5767188b2b76)

According to Github [documentation](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files), we should use it like so: `echo "{environment_variable_name}={value}" >> "$GITHUB_ENV"`